detection_rules/cli_utils.py

```diff
@@ -18,13 +18,11 @@
 from .attack import matrix, tactics, build_threat_map_entry
 from .rule import TOMLRule, TOMLRuleContents
 from .rule_loader import (RuleCollection,
-                          DEFAULT_PREBUILT_RULES_DIR,
-                          DEFAULT_PREBUILT_BBR_DIR,
+                          DEFAULT_PREBUILT_RULES_DIRS,
+                          DEFAULT_PREBUILT_BBR_DIRS,
                           dict_filter)
```

Comment:
a side thought -- @Mikaayenson @eric-forte-elastic how do you guys feel about going `black`-enforced styling? I see we're using flake8 linting but I think it might be beneficial to add at least `black` in the mix. TDS pipeline [runs](https://buildkite.com/elastic/protections-cloud/builds/15030#018b3f9a-1b49-4613-bb48-0c75caef334e) isort + flake8 + black + pyright, but it might be an overkill for us.

Reply:
Generally speaking, I think we do want black in detection rules. We use black in most of our other repos, it would make sense to also do that here. But definitely something I think we would want to wait until DAC is merged back to main, or do it in main and merge into DAC, as it would make a lot of changes in many of the files.

Reply:
++ we also started exploring ruff in our other repos.

Reply:
We should look at this in a different issue.